crates/storage/src/column.rs

```diff
@@ -67,6 +67,9 @@ pub enum Column {
     UploadedBytecodes = 19,
     /// See [`Blobs`](fuel_vm_private::storage::BlobData)
     Blobs = 20,
+    /// See [`FullFuelBlocks`](crate::tables::FullFuelBlocks)
+    /// we don't use this table at the moment, only used for benchmarks
+    FullFuelBlocks = 22,
```

Comment:
Benchmarks shouldn't affect the columns that are used in the real logic. RocksDB allows opening it with a custom set of columns.

Reply:
continuing work in #2159

Reply:
precisely in https://github.com/FuelLabs/fuel-core/pull/2159/files#diff-0e430672a8d94fbbca929879774abd2f09dd2a86eca77c32c77615e100b51831R14-R34